src/components/scenes/PluginViewScene.js

```diff
@@ -246,10 +294,14 @@ class PluginView extends React.Component<PluginProps, PluginState> {
           allowUniversalAccessFromFileURLs
           onMessage={this._onMessage}
           onNavigationStateChange={this._onNavigationStateChange}
-          originWhitelist={['file://', 'https://']}
+          originWhitelist={['file://', 'https://', 'http://', 'edge://']}
           ref={this._setWebview}
           scalesPageToFit={contentScaling}
           source={this._renderWebView()}
+          setWebContentsDebuggingEnabled={true}
```

Comment:
What is the ramification of leaving this on in production?

Reply:
No ramification. It enables you to use safari to debug the webview react native code etc.. used it to see console logs from the plugin that do not surface in react native debugger. removed it anyway.